p4_constraints/BUILD.bazel

```diff
@@ -1,6 +1,7 @@
+load("@com_google_protobuf//bazel:cc_proto_library.bzl", "cc_proto_library")
 load("@rules_cc//cc:cc_library.bzl", "cc_library")
 load("@rules_cc//cc:cc_test.bzl", "cc_test")
-load("@rules_cc//cc:defs.bzl", "cc_proto_library")
+# GOOGLE ONLY (DO NOT REMOVE): load("//third_party/protobuf/bazel:cc_proto_library.bzl", "cc_proto_library")
```

Comment:
I would guess we don't need this since we put the same thing (after search/replace) above, but fine to fix in post ;).